packages/core-p2p/lib/monitor.js

```diff
@@ -103,47 +103,33 @@ class Monitor {
     }
 
     if (networkStart) {
-      logger.warn(
-        'Skipped peer discovery because the relay is in genesis-start mode.',
-      )
+      logger.warn('Skipped peer discovery because the relay is in genesis-start mode.')
       return
     }
 
     if (this.config.disableDiscovery) {
-      logger.warn(
-        'Skipped peer discovery because the relay is in non-discovery mode.',
-      )
+      logger.warn('Skipped peer discovery because the relay is in non-discovery mode.')
       return
     }
 
     try {
-      if (process.env.ARK_ENV !== 'test') {
```

Comment:
I think we need to keep this check.

Reply:
Unit tests run with the testnet config which has 0/1 peers so this will loop forever, that is why the check exists there.

Reply:
I did not mean to disable the check. I noticed that there is such a check already in the very beginning of `updateNetworkStatus()`: `if (test) return`. Which means there is no need to do this check a second time. This is why I removed it.

Now I noticed that the "is test" check at the beginning of `updateNetworkStatus()` is `process.env.NODE_ENV === "test"` whereas later we checked `process.env.ARK_ENV`. I think this must have been a type and it should be `ARK_ENV`. Updated in the patch.